posthog/ai/openai/openai_converter.py

```diff
@@ -55,17 +109,9 @@ def format_openai_response(response: Any) -> List[FormattedMessage]:
                     )
 
                 if hasattr(choice.message, "tool_calls") and choice.message.tool_calls:
-                    for tool_call in choice.message.tool_calls:
-                        content.append(
-                            {
-                                "type": "function",
-                                "id": tool_call.id,
-                                "function": {
-                                    "name": tool_call.function.name,
-                                    "arguments": tool_call.function.arguments,
-                                },
-                            }
-                        )
+                    content.extend(
+                        _format_tool_calls_to_content(choice.message.tool_calls)
```

Comment:
This fixes it for the chat completions response API format. What about the responses API format?